### ISSUE TITLE
ImportKiller doesn't work with Python 3.3

### ISSUE THREAD

**Issue by jaraco** [jaraco/keyring#102]
ImportKiller doesn't work with Python 3.3, which makes the SecretStrorage test fail:

```
Python 3.3.1 (default, Apr 17 2013, 22:32:14) 
[GCC 4.7.3] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from util import ImportKiller
>>> with ImportKiller('secretstorage'):
...     import secretstorage
... 
>>> 
```

---
- Bitbucket: https://bitbucket.org/kang/python-keyring-lib/issue/102
- Originally reported by: Dmitry Shachnev
- Originally created at: 2013-06-08T12:54:57.385

**Comment by jaraco** [comment]
I've added a fix for this issue to my pull request #31.

---

Original comment by: Dmitry Shachnev

**Comment by jaraco** [comment]
Fix ImportKiller not working with Python 3 (Fixes #102)

In Python 3, initial sys.meta_path is not empty, so custom importer modules
should be prepended, not appended.

→ <<cset b03818afe854>>

---

Original comment by: Jason R. Coombs
